cmd/krel/cmd/testgrid.go

```diff
@@ -215,7 +229,7 @@ func generateIssueComment(testgridJobs []TestGridJob, opts *TestGridOptions) {
 				output = append(output,
 					fmt.Sprintf(
 						"<details><summary><tt>%[1]s</tt> %[2]s <a href=\"%[5]s/%[3]s#%[4]s&width=30\">%[3]s#%[4]s - TestGrid</a></summary><p>\n"+
-							"![%[3]s#%[4]s](%[6]s)\n"+
+							"\n![%[3]s#%[4]s](%[6]s)\n\n"+
```

Comment:
@xmudrii this fix the issue you have this morning

Reply:
Yes, that fixed the issue today.